app/controllers/messages_controller.rb

```diff
@@ -6,10 +6,6 @@ class MessagesController < ApplicationController
     controller.ensure_logged_in t("layouts.notifications.you_must_log_in_to_send_a_message")
   end
 
-  before_filter do |controller|
-    controller.ensure_authorized t("layouts.notifications.you_are_not_authorized_to_do_this")
-  end
-
```

Comment:
In ``MessagesController`` it's the `ìs_participant?``on row 14. In ``PeopleController`` only ``@current_user`` is referenced in the resource methods that the filter used to apply to.